lib/phobos_db_checkpoint/events_api.rb

```diff
@@ -72,5 +72,10 @@ class EventsAPI < Sinatra::Base
         .offset(offset)
         .to_json
     end
+
+    get "/#{VERSION}/error_events" do
+      content_type :json
+      PhobosDBCheckpoint::Failure.all.to_json
```

Comment:
I think this should be paginated, it's easy to end up with thousands of failures when something happen